client/src/app/components/CreateProject/styles.css

```diff
@@ -40,20 +40,30 @@
 }
 
 .form-wrapper button {
-  padding: 5px;
-  margin: 10px;
+  border: none;
+  height: 35px;
+  width: 75px;
+  padding: 8px;
+  margin: 2px;
+  background: #ec5252;
+  border: 1px solid #e8e9eb;
```

Comment:
You have on the first line border: none. Is that a mistake?

Reply:
Should be border: none. The other border property was the mistake!